### PR TITLE
Update BasicHttpBinding.xml

### PR DESCRIPTION
**Pull request by rschindhelm**
## Summary

Added bullet points in remarks section of MaxReceivedMessageSize - easier on the eyes.

### CODE REVIEW

File: xml/System.ServiceModel/BasicHttpBinding.xml [hunk 1]
```diff
@@ -641,10 +641,9 @@ Extensive allocation from the CLR garbage heap is an indication that the message
 
 The value of the MaxBufferSize property and its significance is different depending on whether the messages are being buffered or streamed on the channel through which they are being received:
 
-For buffered transports, (<xref:System.ServiceModel.BasicHttpBinding.TransferMode> set to <xref:System.ServiceModel.TransferMode.Buffered>). 
-This value is always equal to that specified by the <xref:System.ServiceModel.BasicHttpBinding.MaxReceivedMessageSize>.
+- For buffered transports, (<xref:System.ServiceModel.BasicHttpBinding.TransferMode> set to <xref:System.ServiceModel.TransferMode.Buffered>), this value is always equal to that specified by the <xref:System.ServiceModel.BasicHttpBinding.MaxReceivedMessageSize>.
```
Comment (rpetrusha) [inline xml/System.ServiceModel/BasicHttpBinding.xml]:
```suggestion
- For buffered transports (<xref:System.ServiceModel.BasicHttpBinding.TransferMode> set to <xref:System.ServiceModel.TransferMode.Buffered?displayProperty=nameWithType>), this value is always equal to that specified by the <xref:System.ServiceModel.BasicHttpBinding.MaxReceivedMessageSize> property.
```

File: xml/System.ServiceModel/BasicHttpBinding.xml [hunk 2]
```diff
@@ -641,10 +641,9 @@ Extensive allocation from the CLR garbage heap is an indication that the message
...
 
-For streamed transports, (<xref:System.ServiceModel.BasicHttpBinding.TransferMode> set to <xref:System.ServiceModel.TransferMode.Streamed>), only the SOAP headers must be buffered to generate the message. 
+- For streamed transports, (<xref:System.ServiceModel.BasicHttpBinding.TransferMode> set to <xref:System.ServiceModel.TransferMode.Streamed>), only the SOAP headers must be buffered to generate the message.
```
Comment (rpetrusha) [inline xml/System.ServiceModel/BasicHttpBinding.xml]:
```suggestion
- For streamed transports (<xref:System.ServiceModel.BasicHttpBinding.TransferMode> set to <xref:System.ServiceModel.TransferMode.Streamed?displayProperty=nameWithType>), only the SOAP headers must be buffered to generate the message. 
```

**Comment by mairaw** [comment]
@rpetrusha I've applied your comments. Please take a look.

**Comment by rpetrusha** [comment]
I'll merge your PR now, @rschindhelm. The updated documentation should be live in the next day or two.

**Pull request merged**